app/src/main/java/org/oppia/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -31,15 +32,17 @@ class DragAndDropSortInteractionViewModel(
     indexTo: Int,
     adapter: RecyclerView.Adapter<RecyclerView.ViewHolder>
   ) {
-    val item = choiceItems[indexFrom]
-    choiceItems.removeAt(indexFrom)
-    choiceItems.add(indexTo, item)
-    if (allowMultipleItemsInSamePosition) {
-      choiceItems[indexFrom].itemIndex = indexFrom
-      choiceItems[indexTo].itemIndex = indexTo
+    if (indexFrom == NO_ITEM && indexTo == NO_ITEM && allowMultipleItemsInSamePosition) {
```

Comment:
Can you add a comment to explain the indexFrom/indexTo logic here & below?

Reply:
done